docs/en/reference/configuration.rst

```diff
@@ -16,8 +15,7 @@ Define the following requirement in your ``composer.json`` file:
     }
 
 Then call ``composer install`` from your command line. If you don't know
-how Composer works, check out their `Getting Started
-<http://getcomposer.org/doc/00-intro.md>`_ to set up.
+how Composer works, check out their `Getting Started <http://getcomposer.org/doc/00-intro.md>`_ to set up.
```

Comment:
https://getcomposer.org/doc/00-intro.md

Reply:
I used https in the other composer-links in the docs too.